addons/l10n_es_edi_facturae/i18n/es.po

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server saas~16.3\n"
+"Project-Id-Version: Odoo Server 17.2alpha1+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2024-01-02 13:00+0000\n"
-"PO-Revision-Date: 2024-01-02 14:07+0100\n"
+"POT-Creation-Date: 2024-01-30 09:04+0000\n"
+"PO-Revision-Date: 2024-01-30 09:04+0000\n"
```

Comment:
Note that it is generally good to avoid changing the date in order to avoid possible conflicts with other PRs changing the translations and also changing the date
No one cares about the date :wink: